CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [1.11.0] - 2024-04-26
```

Comment:
@ylyangtw is right about the date. Could you change that to next Thursday?

Reply:
I think we decided to change this to 4/30 next Tuesday instead.

Reply:
updated. :)